apps/roam/src/components/GitHubSync.tsx

```diff
@@ -944,36 +945,6 @@ const initializeGitHubSync = async (onloadArgs: OnloadArgs) => {
                     },
                   },
                 } as Field<CustomField>,
```

Comment:
I think you should be able to remove the entire config page observer, no?  I don't believe we are using it anymore